charts/fluid/fluid/values.yaml

```diff
@@ -44,32 +44,32 @@ runtime:
     fuse:
       image: registry.cn-shanghai.aliyuncs.com/jindofs/jindo-fuse:3.8.0
     controller:
-      image: fluidcloudnative/jindoruntime-controller:v0.8.0-90d2f18
+      image: fluidcloudnative/jindoruntime-controller:v0.8.0-c0c56a2
     init:
       portCheck:
         enabled: false
```

Comment:
Should we move the `runtime.jindo.init.portCheck.enabled` to make it a global scope option for all the runtime just like the ` runtime criticalFusePod` ? For example, move it to `runtime.options.initPortCheck` ?

Reply:
I think it's a good suggestion! We can do it in another PR. Thanks very much.

Reply:
FYI, https://github.com/fluid-cloudnative/fluid/issues/1505